go/client/cmd_verify.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"io/ioutil"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+	rpc "github.com/keybase/go-framed-msgpack-rpc"
+)
+
+func NewCmdVerify(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "verify",
+		Usage: "Verify message or file signatures for keybase users",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdVerify{Contextified: libkb.NewContextified(g)}, "verify", c)
+		},
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "d, detached",
+				Usage: "Specify a detached signature file.",
+			},
+			cli.StringFlag{
+				Name:  "i, infile",
+				Usage: "Specify an input file.",
+			},
+			cli.StringFlag{
+				Name:  "m, message",
+				Usage: "Provide the message to verify on the command line.",
+			},
+			cli.StringFlag{
+				Name:  "S, signed-by",
+				Usage: "Assert signed by the given user (can use user assertion format).",
+			},
+		},
```

Comment:
have  `-o, --outfile` option.  And also a `--no-output` flag to keep the current behavior.